app/assets/javascripts/Components/grades_summary_display.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import {render} from 'react-dom';
+import { CourseSummaryTable } from './course_summaries_table';
+import { DataChart } from './Helpers/data_chart';
+
+class GradesSummaryDisplay extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      columns: [],
+      data: [],
+      loading: true,
+      datasets: [],
+      labels: [],
+      xLabel: '',
```

Comment:
Rename this `title` as having `labels` and then also` x / y labels` in the state can be confusing to future editors.